src/Workspaces/CSharp/Portable/CodeGeneration/ParameterGenerator.cs

```diff
@@ -148,6 +148,11 @@ internal static ParameterSyntax GetParameter(IParameterSymbol p, CodeGenerationO
             }
 
             var attributes = parameter.GetAttributes();
+            if (parameter.Type.IsValueType)
```

Comment:
❔ What about nullable value types, such as `int?`?

Reply:
This covers nullable value types, since `System.Nullable<T>` itself is a value type.

Reply:
Do the flow analysis attributes apply to nullable value types?

Reply:
I'm pretty sure they only work for reference types. `int?` is just `Nullable<int>`, which is a struct. You can't null deref them directly, instead this is handled by throwing an exception if `Nullable<int>.HasValue` is false.

Reply:
> I'm pretty sure they only work for reference types.

That was my original idea. Tag @sharwell for the value type designs.

Reply:
Confirmed with @333fred  that flow analysis DOES work on `Nullable<T>`, including with attributes